C-SharpBasic/CSharp-Assignment-2/GenericClass.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace C_Sharp
+{
+    internal class GenericClass<INT, STRING>
```

Comment:
how generic classes are declared?

Reply:
In C# generic classes are defined using type parameter brackets after class name.